### ISSUE TITLE
Full screen gliches on Ubuntu 18.04 in Gnome Shell

### ISSUE THREAD

**Issue by berkes** [hovancik/stretchly#295]
### Prerequisites

- [x] I'm using latest version: v 0.17.0

### Description

When switching to "full-screen", the windows remain in a partial square and text, progress-bar and skip-links are moved out of view. See screenshot: 

![screenshot](https://user-images.githubusercontent.com/77059/45306013-e2495000-b51b-11e8-8c12-a097049215d5.png)

Operating system: Ubuntu 18.04.1 LTS 
Dual Screen, Intel® HD Graphics 530 (Skylake GT2)

### Steps to Reproduce

1. In settings: Reset all settings to "Default"
2. In settings: Choose "breaks are full screen" from third window.
3. Wait for next break to op-up.

Actually having dual monitors or a single monitor does not seem to affect the bug.

**Expected behavior:** Both monitors should be covered entirely, text nicely centered

**Actual behavior:** Text and interaction becomes unaccessible and unreadable.

**Reproduces how often:** Always

### Additional Information

Using gnome-shell, as shipped with 18.04. Not customized nor configured overy. Just some minor settings using the common tools (not some -tweak tool, just the settings): things like localisation set to my region and so.

**Comment by hovancik** [comment]
closing as duplicate of #186 (still waiting on electron fix)
